content/config/containers/logging/configure.md

```diff
@@ -149,9 +149,11 @@ STDERR or STDOUT streams block.
 The `mode` log option controls whether to use the `blocking` (default) or
 `non-blocking` message delivery.
 
-The `max-buffer-size` log option controls the size of the buffer used for
-intermediate message storage when `mode` is set to `non-blocking`. `max-buffer-size`
-defaults to 1 megabyte.
+The `max-buffer-size` controls the size of the buffer used for
+intermediate message storage when `mode` is set to `non-blocking`.
+The default is `1m` meaning 1 megabyte (1 million bytes).
+See [here](https://pkg.go.dev/github.com/docker/go-units#FromHumanSize) for the allowed format strings,
```

Comment:
I expect the linter may complain about using `here` as caption for the link; we generally try to use a more descriptive title for the link itself (also to help discoverability of the link).

Reply:
Agreed, I have updated